code/modules/atmospherics/machinery/components/gas_recipe_machines/atmos_machines_recipes.dm

```diff
@@ -115,18 +115,18 @@ GLOBAL_LIST_INIT(gas_recipe_meta, gas_recipes_list())
 	min_temp = 10000
 	max_temp = 30000
 	reaction_type = ENDOTHERMIC_REACTION
-	energy_release = 650000
-	requirements = list(/datum/gas/carbon_dioxide = 10000)
+	energy_release = 9500000
+	requirements = list(/datum/gas/carbon_dioxide = 1500)
 	products = list(/obj/item/stack/sheet/mineral/diamond = 1)
 
 /datum/gas_recipe/crystallizer/plasma_sheet
 	id = "plasma_sheet"
 	name = "Plasma sheet"
-	min_temp = 100
-	max_temp = 140
-	reaction_type = ENDOTHERMIC_REACTION
-	energy_release = 15000
-	requirements = list(/datum/gas/plasma = 25)
+	min_temp = 10
+	max_temp = 20
+	reaction_type = EXOTHERMIC_REACTION
+	energy_release = 3500000
+	requirements = list(/datum/gas/plasma = 450)
 	products = list(/obj/item/stack/sheet/mineral/plasma = 1)
```

Comment:
Gonna ask again since I'm not sure it ever got answered, what's the plasma gas return on this?

Reply:
for each unit of reagent, it spawns 1 mole of gas, each sheet is 20 unit

Reply:
Sweet